crates/sui-config/src/genesis.rs

```diff
@@ -128,9 +217,8 @@ impl Builder {
     //     self
     // }
 
-    //TODO actually use the validators added to genesis
```

Comment:
The issue is that today the validator set is not being managed on-chain, its only being managed via config. While this hasn't fully changed yet, after this PR the validator set is now inside the "genesis" blob that we plan to disseminate to folks running their own nodes (since its not data that they should actually change but should be fixed for everyone)